geograypher/meshes/meshes.py

```diff
@@ -123,6 +123,16 @@ def __init__(
         )
         # Load the texture
         self.logger.info("Loading texture")
+        # load IDs_to_labels
+        # if IDs_to_labels not provided, check the directory of the mesh and get the file if found
+        if IDs_to_labels == None and isinstance(mesh, PATH_TYPE.__args__):
```

Comment:
Using the `is None` idiom is preferred in Python, though this is sufficient in most cases. https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none